addons/storyshots/storyshots-core/src/api/index.ts

```diff
@@ -21,8 +21,13 @@ function callTestMethodGlobals(
   });
 }
 
-const isDisabled = (parameter: any) =>
-  parameter === false || (parameter && parameter.disable === true);
+const isDisabled = (parameter: any, suite: string) =>
+  parameter === false ||
+  (parameter && parameter.disable === true) ||
+  (parameter &&
+    parameter.disabledSuites &&
+    (parameter.disabledSuites === suite ||
```

Comment:
Is there a reason to support the case where `disabledSuites` is a string?

the `parameters === false` case is also kinda strange but maybe it's preexisting.

I think we could simplify this and write is as

```
parameter === false ||
(parameter && (parameter.disable === true || parameter.disabledSuites.includes(suite) )
```

I don't think we need to test for isArray, just let it fail with an exception ( probably that 'includes' isn't a method or the like)

Reply:
The === false thing is documented and preexisting. it’s kinda what inspired
the string option just for ease of use since there was precedence for type
loosely gooseyness. I’m happy to remove it (and the array is array check)... the func it probably longer than it needs to be